core-text/src/font.rs

```diff
@@ -944,7 +944,8 @@ fn out_of_range_variations() {
     let small = new_ui_font_for_language(kCTFontSystemDetailFontType, 19., None);
 
     let axes = small.get_variation_axes();
-    if macos_version() < (10, 12, 0) {
+    let version = dbg!(macos_version());
```

Comment:
I think you can remove `dbg!(...)` here. It doesn't add too much when you aren't working on this particular section of the code.

Reply:
There's a bunch of instances of `dbg!()` already in this test, which was really useful when this test "suddenly" started failing due to me being on a newer OS version, so I'd like to keep this around (which helps see on test failure what case is being hit).

(In general, I think it's pretty common to leave `println!()` or `dbg!()` calls in test code, since you'll usually only see it on test failure anyway.)